src/app/App.tsx

```diff
@@ -100,7 +105,9 @@ export class App extends React.Component<Props> {
       }
     })
 
-    Linking.addEventListener('url', this.handleOpenURL)
+    Linking.addEventListener('url', async (event) => {
+      return this.handleOpenURL(event, OVERRIDE_DEEP_LINK_SECURE_ORIGIN)
+    })
```

Comment:
We shouldn't override this for this test. It's a security feature and we should use a different deep link instead for testing.
Would be nice to actually test this security feature via the e2e test too.
i.e. using calling `openScreen` deeplink will not open said screen.